src/lang/es-MX.json

```diff
@@ -57,12 +58,13 @@
 	"alt-text.resource-suggest-edits-thank-you": "Gracias",
 	"announcement.border-closure-header": "Las fronteras de los Estados Unidos están cerradas debido al coronavirus:",
 	"announcement.border-closure-full": "A partir de agosto de 2021, se ha restablecido la política \"Permanecer en México\". Lxs solicitantes de asilo se ven obligadxs a esperar sus procedimientos judiciales en México. Esta política puede cambiar y actualizaremos la información según sea necesario. Los catálogos de México y Estados Unidos estarán disponibles en inglés y español a finales del año, con todos los demás idiomas disponibles a través de Google Translate.",
+	"announcement.localisation": "Los Catálogos de México y los EE.UU estarán disponibles en inglés y español para finales de año, con los demás idiomas disponibles a través de Google Translate.",
+	"app.choose-different-country": "Elegir otro país",
 	"app.asylum-connect-catalog": "Catálogo de AsylumConnect",
 	"app.inactivity-sign-in": "Debido a inactividad, por favor inicia una sesión para confirmar tu identidad",
-	"app.choose-different-country": "Elegir otro país",
+	"app.welcome": "¡Bienvenidx al Catálogo de México de AsylumConnect!",
```

Comment:
@trigal2012 just a note that we'd like to keep these keys listed in alphabetical order. Thanks!

Reply:
yeah, I noticed some of the language files had keys in a different order from one another making it difficult to chase down an issue I had. I made the order match across all files. I can do a quick, separate, PR to get everything alphabetized across all of the files.